src/plugins/blocksmith/resolver.js

```diff
@@ -198,7 +198,7 @@ export const getModalComponent = (message) => {
  *
  * @returns {?Component} a React component that is intended to go inside the DraftJs editor
  */
-export const getPlaceholder = (type, decorator, props = {}) => {
+export const getPlaceholder = (type, props = {}) => {
```

Comment:
@efjacobson - please update the DocBlock as well.

Reply:
@arunkarnati done